src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpTraceExporterHelperExtensions.cs

```diff
@@ -106,7 +107,8 @@ public static TracerProviderBuilder AddOtlpExporter(this TracerProviderBuilder b
         {
             exporterOptions.TryEnableIHttpClientFactoryIntegration(serviceProvider, "OtlpTraceExporter");
 
-            BaseExporter<Activity> otlpExporter = new OtlpTraceExporter(exporterOptions, sdkLimitOptions);
+            PersistentBlobProvider persistentBlobProvider = serviceProvider?.GetService<PersistentBlobProvider>();
+            BaseExporter<Activity> otlpExporter = new OtlpTraceExporter(exporterOptions, sdkLimitOptions, null, persistentBlobProvider);
```

Comment:
Should we just add `PersistentBlobProvider` to options and not passing through constructor?

Reply:
I believe we don't want to add more things to the options class.

Reply:
If you want you could add it on there as internal just to clean up all the code needing to pass it around.